hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/fsck/TestMissingContainerTask.java

```diff
@@ -90,15 +102,19 @@ public void testRun() throws Exception {
     missingContainerTask.start();
 
     LambdaTestUtils.await(6000, 1000, () ->
-        (missingContainersTableHandle.findAll().size() == 1));
+        (missingContainersTableHandle.findAll().size() == 2));
     all = missingContainersTableHandle.findAll();
-    Assert.assertEquals(3, all.get(0).getContainerId().longValue());
-
+    // Container IDs 2 and 3 should be present in the missing containers table
+    Set<Long> missingContainerIDs = Collections.unmodifiableSet(
+        new HashSet<>(Arrays.asList(2L, 3L))
+    );
+    Assert.assertTrue(all.stream().allMatch(record ->
+        missingContainerIDs.stream().anyMatch(
```

Comment:
Nit. This may be replaced with something like 

`all.forEach(r -> Assert.assertTrue(missingContainerIDs.contains(r.getContainerId())));`